addons/event/models/event_event.py

```diff
@@ -112,9 +112,10 @@ def _get_default_stage_id(self):
         'res.partner', string='Organizer', tracking=True,
         default=lambda self: self.env.company.partner_id,
         domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
-    event_type_id = fields.Many2one('event.type', string='Category', ondelete='set null')
+    event_type_id = fields.Many2one('event.type', string='Template', ondelete='set null')
     color = fields.Integer('Kanban Color Index')
     event_mail_ids = fields.One2many('event.mail', 'event_id', string='Mail Schedule', copy=True)
+    tag_ids = fields.Many2many('event.tag', string="Tags")
```

Comment:
You will have to make a computed editable once onchange -> store is merged (which is normally soon).